nautobot/dcim/views.py

```diff
@@ -1666,6 +1667,8 @@ def get_extra_context(self, request, instance):
         context = super().get_extra_context(request, instance)
         if self.action == "retrieve":
             context["network_driver_tool_names"] = instance.fetch_network_driver_mappings()
+        if self.action in ["create", "update"]:
+            context["network_driver_names"] = sorted(get_all_network_driver_mappings().keys())
```

Comment:
Where's this additional context data used? I don't see a change to a template to make use of it?

Reply:
It's a bug fix. I think we thought it was fixed... but it wasn't